test/system/160-volumes.bats

```diff
@@ -319,5 +319,52 @@ EOF
     is "$output"  "" "no more volumes to prune"
 }
 
+# Named volumes copyup
+@test "podman volume create copyup" {
+    myvolume=myvol$(random_string)
+    mylabel=$(random_string)
+
+    # Create a named volume
+    run_podman volume create $myvolume
+    is "$output" "$myvolume" "output from volume create"
+
+    # Confirm that it shows up in 'volume ls', and confirm values
+    run_podman volume ls --format json
+    tests="
+Name           | $myvolume
+Driver         | local
+NeedsCopyUp    | true
+"
+    parse_table "$tests" | while read field expect; do
+        actual=$(jq -r ".[0].$field" <<<"$output")
+        is "$actual" "$expect" "volume ls .$field"
+    done
+
+    run_podman run --rm --volume $myvolume:/vol $IMAGE true
+    # Confirm that it shows up in 'volume ls', and confirm values
+    run_podman volume ls --format json
+    tests="
+Name           | $myvolume
+Driver         | local
+NeedsCopyUp    | true
+"
+    run_podman run --rm --volume $myvolume:/etc $IMAGE ls /etc/passwd
+    # Confirm that it shows up in 'volume ls', and confirm values
+    run_podman volume ls --format json
+    tests="
+Name           | $myvolume
+Driver         | local
+NeedsCopyUp    | false
+"
```

Comment:
This looks like a NOP. Copy/paste artifact maybe?

Reply:
No, second one has NeedsCopyUp set to False.

Reply:
...but there's no `parse_table`. These last two definitions of `test` are not actually used.

Signing out for the day since, as I just learned, today is a holiday.

Reply:
Went with inspect check.